integrations/observability/prometheus-operator/install.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+# Copyright KubeArchive Authors
+# SPDX-License-Identifier: Apache-2.0
+
+set -o errexit
+set -o xtrace
+
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+NAMESPACE="prometheus-operator"
+
+helm repo add prometheus-community https://prometheus-community.github.io/helm-charts
+helm repo update
+helm upgrade --install --wait --create-namespace --namespace ${NAMESPACE} \
+	prometheus prometheus-community/kube-prometheus-stack \
+	--set prometheus.prometheusSpec.serviceMonitorSelectorNilUsesHelmValues=false
```

Comment:
Should we add `helm` as a requirement in the README now that it's not a requirement for KubeArchive per se?

Reply:
It is specified in the `DEVELOPMENT.md` as a requirement for development of KubeArchive, so I don't think we need to point out that we are using `helm` here.